doxia-modules/doxia-module-markdown/pom.xml

```diff
@@ -51,19 +51,57 @@ under the License.
     </contributor>
   </contributors>
 
+  <properties>
+    <flexmark.version>0.42.14</flexmark.version>
```

Comment:
I think we should use the camelCase approach for version properties everywhere now: `flexmarkVersion`

Reply:
master is at https://github.com/apache/maven-doxia/blob/master/doxia-modules/doxia-module-markdown/pom.xml#L67 `0.62.2`, 1.11.x requires a different version though

Reply:
done the camelCase, you're right, it's what we do for parent POMs
on version, yes, the current PR is for Doxia 1.x, Doxia 2.x will get another version

Reply:
If for 1.x then then PR is wrong. IT targets master.